### ISSUE TITLE
Navigator breaking chart positioning.

### ISSUE THREAD

**Issue by henrikej** [highcharts/highcharts-react#78]
When adding the Navigator from HighStock, removing then adding a series, breaks its positioning.
The data needs to be in separate series and split up.

``` 
import Highcharts from "highcharts/highstock";
import HighchartsReact from "highcharts-react-official";

chartOptions = () => {
        this.props.observationData.rangeSelector =  {
            selected: 1,
            enabled: false
        };
        this.props.observationData.title = {
            text: ''
        };
          this.props.observationData.navigator = { 
            navigator: true
        };
![screenshot from 2018-12-13 13-44-40](https://user-images.githubusercontent.com/5745941/49939591-907ae200-fedd-11e8-98c6-9197d731e48a.png)

        this.props.observationData.chart =  {
            style: this.props.classes.chartStyle
        };
        this.props.observationData.credits =  {
            enabled: false
        };
        this.props.observationData.tooltip =  {
            shared: true,
            crosshairs: true,
            split: false
        };
        return this.props.observationData;
    };

 <HighchartsReact
highcharts={Highcharts}
options={this.chartOptions()}
 />
```

This has been tried with both adding the navigator: true, and using constructorType={'stockChart'} 

![screenshot from 2018-12-13 13-44-40](https://user-images.githubusercontent.com/5745941/49939640-b2746480-fedd-11e8-90e9-7dbb73d74409.png)
Adding two charts works as intended initially 
![screenshot from 2018-12-13 13-45-28](https://user-images.githubusercontent.com/5745941/49939595-94a6ff80-fedd-11e8-8f09-f707e4e520b7.png)
When removing one chart, is scales correctly 
![screenshot from 2018-12-13 13-45-45](https://user-images.githubusercontent.com/5745941/49939602-9670c300-fedd-11e8-877c-b56d822f51cf.png)
When again adding the series to the dataset, it becomes placed under the navigator.(this should end 
up looking like figure 1)
![screenshot from 2018-12-13 13-58-45](https://user-images.githubusercontent.com/5745941/49940233-45fa6500-fedf-11e8-85a6-c119b2f1292e.png)
It only seems like this occurs with the second element to be placed.
The spacing of the charts are calculated by:
``` 
   height: (height - 2) + "%",
                lineWidth:2,
                opposite:false,
                top:topVal + "%",
                offset:0,
                labels:{align:"right",x:-3},
                title:{text:element}});
``` 
which translates to:

``` 
1:
  height: "31%"
  labels: {…}
  lineWidth: 2
  offset: 0
  opposite:false
  title:{…}
  top: "0%
2: (element that gets missplaced)
  height: "31%"
  labels: {…}
  lineWidth:2
  offset: 0
  opposite:false
  title:{…}
  top:"33%"
3:
  [...]
  top: "66%"
```

**Comment by ppotaczek** [comment]
Hi @henrikej,

Thank you for reporting!

I reproduced this problem without React (http://jsfiddle.net/BlackLabel/gnrqmjLf/) and I reported it on 'highcharts' repository: https://github.com/highcharts/highcharts/issues/9671

Best regards!

**Comment by akhtariali** [comment]
@ppotaczek Still having this issue on `highcharts-react-official : ^3.0.0` and `highcharts : ^8.2.2`. Only the series with the index of 1 ends up in the navigator box! 👀

**Comment by ppotaczek** [comment]
Hi @akhtariali,

The issue should be fixed in Highchart v9.0.1. It is closed because it is caused by Highcharts not by `highcharts-react-official`.

**Comment by akhtariali** [comment]
@ppotaczek Pretty sure I've experienced the issue when loading the charts inside a `.map`. I moved on and replaced the functionality using the pan and zoom feature. So it's not a priority for my team right now. Will report the issue on HighCharts main repo if I faced it again.
